tasks/Task.cpp

```diff
@@ -70,21 +70,16 @@ void Task::updateHook()
     motionCommand.rotation    = 0;
     motionCommand.heading     = 0;
 
-    if( _robot_pose.readNewest( rbpose ) == RTT::NoData)
+    Eigen::Affine3d robot2map;
+    if(!_robot2map.get(base::Time::now(), robot2map, false))
```

Comment:
You are interested in the latest transformation here, using base::Time::now does not grantee, 
that you get a pose back. Better, register a callback for the transformation, and save the given value
to a class member.

Reply:
I'm calling robot2map.get() with interpoalation = false, I thought this has the same effect, e.g. it returns the latest received transformation. In what situation will it give no result?

Reply:
You are right, I overlooked the interpolation = false, in this case it will work as intended.

Reply:
Ah okay, thanks for clarification!